ui/src/table.tsx

```diff
@@ -543,26 +579,25 @@ export const
         setGroupByKey(option.key as S)
         initGroups()
       },
-      onFilterChange = React.useCallback((filterKey: S, filterVal: S) => (_e?: React.FormEvent<HTMLInputElement | HTMLElement>, checked?: B) => {
+      onFilterChange = React.useCallback((filterKey: S, filterVal: S, checked?: B) => {
         setSelectedFilters(selectedFilters => {
-          const filters = selectedFilters || {}
-
+          let filters: Dict<S[]> | null = null
+          filters = selectedFilters || {}
```

Comment:
What's the reason for this change?

Reply:
It was probably a mistake during the refactoring. Reverted.